spek-extensions/robolectric/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply from: "$rootDir/gradle/common/junit-jupiter.gradle"
+apply plugin: 'org.jetbrains.dokka'
+
+configurations {
+    provided
```

Comment:
We could use the built-in `compileOnly` configuration.

Reply:
Done.